html_css/css/style.css

```diff
@@ -1082,14 +1187,59 @@ button:active {
   box-shadow: 0px 4px 16px var(--send-button-bg-box-shadow);
 }
 
+.footer {
+  position: relative;
+  overflow: hidden;
+}
+
+.footer-bg-icons-left {
```

Comment:
Too long name. Regulary, 2-3 words is max

Reply:
Done